src/notifications/teams.py

```diff
@@ -0,0 +1,79 @@
+from datetime import datetime
+import logging
+import requests
+from typing import Dict
+import json
+
+from db.models import Site, Victim
+from .source import NotificationSource
+
+class TeamsNotification(NotificationSource):
+
+    def _escape_url(url:str) -> str:
```

Comment:
space between the param name and type (`url: str`) (please do this throughout the file)